### ISSUE TITLE
getStoredState returns transit format string

### ISSUE THREAD

**Issue by viz** [rt2zz/redux-persist-immutable#18]
Is it possible to have `getStoredState` return an restored immutable version of the stored state?

At the moment it's just an unwrapped version of the original `getStoredState` so it returns the raw transit format string.

**Comment by rt2zz** [comment]
yes this is a bug, thanks for finding! just released v4.1.0 with fix
